src/common/models/project.go

```diff
@@ -37,20 +37,21 @@ const (
 
 // Project holds the details of a project.
 type Project struct {
-	ProjectID    int64               `orm:"pk;auto;column(project_id)" json:"project_id"`
-	OwnerID      int                 `orm:"column(owner_id)" json:"owner_id"`
-	Name         string              `orm:"column(name)" json:"name" sort:"default"`
-	CreationTime time.Time           `orm:"column(creation_time);auto_now_add" json:"creation_time"`
-	UpdateTime   time.Time           `orm:"column(update_time);auto_now" json:"update_time"`
-	Deleted      bool                `orm:"column(deleted)" json:"deleted"`
+	ProjectID    int64     `orm:"pk;auto;column(project_id)" json:"project_id" sort:"true"`
```

Comment:
The field with `column` annotation is sortable if no `sort` annotation set by default. So no need to set `sort:"true"`